packages/token-lists/package.json

```diff
@@ -15,24 +15,29 @@
     "checksum:pcs-top-100": "yarn build && node ./dist checksum pancakeswap-top-100",
     "generate:pcs-top-100": "yarn test && yarn build && node ./dist generate pancakeswap-top-100",
     "makelist:pcs-top-100": "yarn checksum:pcs-top-100 && yarn generate:pcs-top-100",
+    "fetch:pcs-top-100": "yarn build && node ./dist fetch",
     "test": "jest"
   },
+  "dependencies": {
+    "@ethersproject/address": "^5.1.0",
+    "@uniswap/token-lists": "^1.0.0-beta.21",
+    "ajv": "6.12.2",
+    "graphql": "^15.5.0",
+    "graphql-request": "^3.4.0"
+  },
   "devDependencies": {
     "@babel/preset-env": "^7.14.1",
     "@babel/preset-typescript": "^7.13.0",
     "@pancakeswap-libs/eslint-config-pancake": "^1.0.1",
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-typescript": "^8.2.1",
     "@types/jest": "^26.0.23",
-    "@types/node": "^15.0.2",
+    "@types/node": "^15.12.0",
     "babel-jest": "^26.6.3",
     "jest": "^26.6.3",
     "rollup": "^2.47.0",
-    "ts-jest": "^26.5.6"
-  },
-  "dependencies": {
-    "@ethersproject/address": "^5.1.0",
-    "@uniswap/token-lists": "^1.0.0-beta.21",
-    "ajv": "6.12.2"
+    "ts-jest": "^26.5.6",
+    "ts-node": "^10.0.0",
+    "typescript": "^4.3.2"
```

Comment:
Shouldn't these be in `devDependencies`?

Reply:
It is in `devDependencies` but the diff' make it hard to see; will also remove a few deps like `typescript` because it's pulled from the root `package.json`

Reply:
Fixed in latest commit.